core/templates/dev/head/pages/exploration_player/input_response_pair_directive.html

```diff
@@ -2,8 +2,8 @@
   <img class="conversation-skin-user-avatar img-circle" ng-src="<[profilePicture()]>" alt="">
   <div ng-if="!data.isHint">
     <div ng-if="getShortAnswerHtml()"
-         popover-placement="bottom" uib-popover-template="'popover/answer'"
```

Comment:
I'm not sure about the earlier expected behavior, f we are going to remove the popover entirely then why have you left the "popover-placement" and "popover-trigger"?

@seanlip are we expecting to have a popover here explaining the long answer?

Reply:
There should always be a way for a student to see the full answer they submitted before, but this could be a popup or a modal (though in the latter case we need to think about how it works on mobile phones). Does that answer your question?

Reply:
Thanks for the reply @seanlip, I agree with your answer.
@MShaker, go through the last few lines of this file and you'll find the template for this popover try fixing this and I'll suggest you test this on a mobile device before the final push. 

Do let me know if you have any question regarding this.

Reply:
@seanlip, so in #5606 an unwanted, empty tooltip was mentioned, but to be clear, the issue is not that the tooltip is unwanted, but rather that it should be displaying the full response?

Reply:
Yes, that is correct! Thank you for checking.